scout/server/blueprints/variants/views.py

```diff
@@ -161,15 +160,17 @@ def variants(institute_id, case_name):
     cytobands = store.cytoband_by_chrom(case_obj.get("genome_build"))
 
     variants_query = store.variants(case_obj["_id"], query=form.data, category=category)
+    query = store.get_query(case_obj["_id"], form.data, None, category)
```

Comment:
Why don't you use the store.variants_count that you created in scout/adapter/mongo/variant.py to do that instead?